test/check-machines-consoles

```diff
@@ -744,6 +745,102 @@ fullscreen=0
         b.wait_visible(".vm-console-vnc canvas")
         self.assertIn("password=on", m.execute(f"grep ^-vnc /var/log/libvirt/qemu/{name}.log"))
 
+        # Change global password, but without restarting the VM. This
+        # means that Cockpit will use the new password when
+        # connecting, but the VM still expects the old.
+        m.execute("sed -i -e 's/vnc_password = \"barfoo\"/vnc_password = \"foobar\"/' /etc/libvirt/qemu.conf")
+
+        # We are still connected, but reconnecting will fail.
+
+        b.click(".consoles-card button:contains(Disconnect)")
+        b.wait_in_text(".consoles-card", "Disconnected")
+
+        # Quite often the reconnection here fails with code 1006
+        # (Abnormal Closure), but retrying will give us the expected
+        # error message, usually on the second try.
+
+        b.click(".consoles-card button:contains(Connect)")
+        for _ in range(5):
+            try:
+                b.wait_in_text(".consoles-card", "Authentication failed")
+                break
+            except testlib.Error:
+                if "Failed to connect" in b.text(".consoles-card"):
+                    print("Unexpected error message. Retrying...")
+                    b.click(".consoles-card button:contains(Retry)")
+                else:
+                    raise
+
+        b.wait_in_text(".consoles-card", "Authentication failed")
+
+    @testlib.skipImage("cockpit-ws too old",
+                       "rhel-9-*", "centos-9-*", "ubuntu-*", "debian-trixie", "opensuse-tumbleweed")
```

Comment:
Is that the `websocket-channel-close-fix` thing?

Reply:
yes.